packages/sdk-react/src/contexts/TurnkeyContext.tsx

```diff
@@ -64,6 +64,15 @@ export const TurnkeyProvider: React.FC<TurnkeyProviderProps> = ({
 
           currentClient = authIframeClient;
         }
+      } else {
```

Comment:
A bit confused by this `else`. Wouldn't this mean we weren't able to get the auth bundle? 

Also should we now add a check for a signing session? (in addition to auth bundle)

Reply:
updated the logic for the getCurrentClient function